lib/ts/recipe/thirdpartypasswordless/recipe.tsx

```diff
@@ -0,0 +1,251 @@
+                          palette: this.config.palette,
+                          preAPIHook: this.config.preAPIHook,
+                          signInAndUpFeature: {
+                              ...this.config.signInUpFeature,
+                              style: this.config.signInUpFeature.providerAndEmailOrPhoneFormStyle,
+                          },
+                          oAuthCallbackScreen: this.config.oAuthCallbackScreen,
+                          useShadowDom: this.config.useShadowDom,
+                          override: {
+                              functions: () => getThirdPartyImpl(this.recipeImpl),
+                              components: this.config.override.components,
+                          },
+                      },
+                      {
+                          emailVerificationInstance: this.emailVerification,
+                      }
+                  );
+    }
+
+    getFeatures = (): RecipeFeatureComponentMap => {
+        let features: RecipeFeatureComponentMap = {};
+
+        if (this.passwordlessRecipe !== undefined) {
+            features = {
+                ...features,
+                ...this.passwordlessRecipe.getFeatures(),
+            };
+        }
+
+        if (this.thirdPartyRecipe !== undefined) {
+            features = {
+                ...features,
+                ...this.thirdPartyRecipe.getFeatures(),
+            };
+        }
+
+        if (this.config.signInUpFeature.disableDefaultImplementation !== true) {
+            const normalisedFullPath = this.config.appInfo.websiteBasePath.appendPath(new NormalisedURLPath("/"));
+            features[normalisedFullPath.getAsStringDangerous()] = {
+                matches: matchRecipeIdUsingQueryParams(this.config.recipeId),
+                component: (prop: any) => this.getFeatureComponent("signInUp", prop),
+            };
+        }
+
+        return {
+            ...features,
+            ...this.getAuthRecipeWithEmailVerificationFeatures(),
+        };
+    };
+
+    getDefaultRedirectionURL = async (context: GetRedirectionURLContext): Promise<string> => {
+        return this.getAuthRecipeWithEmailVerificationDefaultRedirectionURL(context);
+    };
+
+    getFeatureComponent = (
+        componentName: "emailverification" | "signInUp" | "linkClickedScreen",
+        props: any
+    ): JSX.Element => {
+        if (componentName === "signInUp") {
+            return (
+                <AuthWidgetWrapper<GetRedirectionURLContext, PreAPIHookContext, OnHandleEventContext, NormalisedConfig>
+                    authRecipe={this}
+                    history={props.history}>
+                    <SignInAndUp recipe={this} {...props} />
+                </AuthWidgetWrapper>
+            );
+        } else if (componentName === "linkClickedScreen") {
+            if (this.passwordlessRecipe === undefined) {
+                throw new Error("Should not come here...");
```

Comment:
Change the error message - it can come here in case the user wants to embed the link clicked screen, but they have disabled passwordless